src/fiuba/algo3/tp2/juego/Mapa.java

```diff
@@ -28,6 +29,16 @@ public Casillero getCasillero(Posicion unaPosicion) {
         return this.casilleros.get(fila).get(columna);
     }
 
+    public boolean estaAfueraDelMapaLaPosicion(Posicion unaPosicion) {
+        boolean estaAfuera = false;
```

Comment:
este metodo se puede resumir a :
{
int fila = unaPosicion.getY();
int columna = unaPosicion.getX();
return (fila < 0 || columna < 0 ||  fila > (this.filas - 1) || columna > (this.columnas - 1));
}

Reply:
ok